src/Frontend/Page/Idea.hs

```diff
@@ -423,3 +464,13 @@ replyCommentIdea ideaId commentId =
                       comment <- idea ^. ideaComments . at commentId
                       pure $ CommentIdea idea (Just comment))
         (currentUserAddDb $ AddReplyToIdeaComment ideaId commentId)
+
+juryIdea :: ActionM m => AUID Idea -> IdeaJuryResultType -> ServerT (FormHandler JuryIdea) m
+juryIdea ideaId juryType =
+    redirectFormHandler
+        (do 
+            -- FIXME: Run in one transaction
+            idea  <- mquery (findIdea ideaId)
+            topic <- mquery (ideaTopic idea)
+            pure $ JuryIdea juryType idea topic)
```

Comment:
i think you can to this:

```haskell
mquery $ do
    midea <- findIdea ideaId
    mtopic <- maybe (pure Nothing) findTopic idea
    pure $ JuryIdea juryTopy <$> midea <*> mtopic
```

ok, not that much prettier, but one transaction.
```